libslide/src/bignum/add.rs

```diff
@@ -73,6 +74,12 @@ impl ops::Add for Bignum {
         if carry == 1 {
             res_int.push(1);
         }
+        // remove zeros
+        res_int = truncate_zeros(res_int);
+        res_dec = truncate_zeros(res_dec);
+        if res_int.is_empty() && res_dec.is_empty() {
+            is_neg = false;
+        }
```

Comment:
do we need this?

Reply:
it stops the result of -0. We can do a few things here. We can stop it in the to_string method and allow -0 in our code. Or we can remove it in every method.

Reply:
let's just handle it in to string

Reply:
ok. will fix.

Reply:
-0 is also an issue for comparisons, but we can just handle it in there